lib/onboarding.dart

```diff
@@ -0,0 +1,117 @@
+              )),
+      //-----------------------------------------------------------Collaboration
+      PageViewModel(
+        image: Image.asset("assets/images/onboarding2.gif"),
+        title: "Rich collaboration platform",
+        body:
+            "Eureka provides you with every tool to conveniently collaborate on any problem ",
+        decoration: const PageDecoration(
+            //pageColor: Color(0xfffdfafc),
+            ),
+        footer: Text("Choose from Text chat, Video chat, or Live Video Call"),
+      ),
+      //-----------------------------------------------------------Privacy
+      PageViewModel(
+        image: Image.asset("assets/images/privacy.png"),
+        title: "Your Privacy Matters!",
+        body:
+            "All communication happens in the app, so you don't have to worry about giving out your personal information to strangers",
+        decoration: const PageDecoration(
+            //pageColor: Color(0xfffdfafc),
+            ),
+        footer: Text(""),
+      ),
+      //-----------------------------------------------------------Categories
+      PageViewModel(
+        image: Image.asset("assets/images/onboarding1.gif"),
+        title: "Multiple categories",
+        body:
+            "Eureka has several most popular categories for questions and answers",
+        decoration: const PageDecoration(
+          pageColor: Color(0xfffdfafc),
+        ),
+        footer: Text("Lifestyle  Education  Electronics  Household"),
+      ),
+      //-----------------------------------------------------------All set
+      PageViewModel(
+        image: Image.asset("assets/images/onboarding3.gif"),
+        title: "You are all set!",
+        body:
+            "Asking and answering questions has never been so easy and rewarding \n You are ready to join !",
+        decoration: const PageDecoration(
+          pageColor: Color(0xff6DB8FF),
+        ),
+        footer: Text("If you still have questions, visit our FAQ section"),
+      ),
+    ];
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("Eureka"),
+      ),
+      body: IntroductionScreen(
+        globalBackgroundColor: Colors.white,
+        pages: getPages(),
+        showNextButton: true,
+        showSkipButton: true,
+        skip: Text("Skip"),
+        done: Text("Got it "),
+        onDone: () {
+          Navigator.push(
+              context,
+              MaterialPageRoute(
+                  builder: (context) =>
+                      placeholder())); // go to placeholder when done is pressed
+        },
+      ),
+    );
+  }
+}
+
+//placeholder class start
+class placeholder extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Colors.blueAccent,
+      child: Center(
+        child: Text('placeholder'),
+      ),
+    );
+  }
+}
+//placeholder class end

```

Comment:
I realized you had created your file under `lib` but for better organization we create different folders https://medium.com/flutter-community/flutter-code-organization-de3a4c219149 This talks about it more in depth but for us, we have a `components` and `screens` folder right now. Please add this file inside `screens` :)